sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/simulator/simulator.py

```diff
@@ -57,6 +60,7 @@ def _validate_project_config(azure_ai_project: Dict[str, Any]):
             raise ValueError("subscription_id, resource_group_name, and project_name must not be None")
 
     # @monitor_task_simulator
+    @experimental
```

Comment:
if the class is already marked as experimental, you don't need to mark the methods on it as well. It's assumed everything inside is subject to change. Customers sometimes get confused when more than one thing is logged for the same class they are using

Reply:
Removed :)